File: app/src/main/java/com/seniorzhai/dagger2sample/view/imp/IMainView.java
package com.seniorzhai.dagger2sample.view.imp;

import com.seniorzhai.dagger2sample.model.Gank;

import java.util.List;

/**
 * Created by zhai on 16/5/23.
 */

public interface IMainView {
    // 显示Loading
    void showLoading();
    // 隐藏Loading
    void hideLoading();
    // 渲染数据
    void fillData(List<Gank> list);
}

File: app/src/main/java/com/seniorzhai/dagger2sample/view/imp/IMainView.kt
package com.seniorzhai.dagger2sample.view.imp

import android.widget.ImageView
import com.seniorzhai.dagger2sample.model.Gank

/**
 * Created by zhai on 16/5/23.
 */

interface IMainView {

    // 显示Loading
    fun showLoading()

    // 隐藏Loading
    fun hideLoading()

    // 渲染数据
    fun fillData(list: List<Gank>)
}
